### ISSUE TITLE
`input` initial mode unexpected behavior

### ISSUE THREAD

**Issue by Martin1887** [Canop/broot#829]
When the `initial_mode` parameter is set to `input`, after changing the directory the mode is automatically returned to `input`.

Maybe this is the expected behavior, but it seems weird to me pressing `h` to focus the current folder and finding the mode in `input` mode again, or navigating into a folder and finding `j` is searched instead of moving the cursor down.

More than an `initial_mode` this seems to me as a `default_mode`, being `modal` mode to be continuously activated after each operation.

Could adding another parameter `default_mode` or adding another option `input_at_start`  to the `initial_mode` parameter the best alternative? I don't know.

Thanks!

**Comment by Canop** [comment]
> Could adding another parameter default_mode or adding another option input_at_start to the initial_mode parameter the best alternative?

Or just take the previous mode ?

**Comment by Martin1887** [comment]
Yes, I'm currently using the previous mode, but I was not sure this is the desired behavior of the input mode and I think the other type of initial mode could be more comfortable for some people, but I cannot confirm it :-).

**Comment by Martin1887** [comment]
If this is the desired behavior then this issue would be a request more than a bug. I'm comfortable on the modal initial mode, so if this issue is not upvoted it should not have high priority.
